dotnet-desktop-guide/framework/winforms/controls/how-to-display-option-buttons-in-a-menustrip-windows-forms.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "How to: Display Option Buttons in a MenuStrip"
+description: Learn how to display Option buttons which are sometimes called Radio buttons in a MenuStrip or Windows form. 
```

Comment:
Learn how to display Option buttons, which are sometimes called Radio buttons, in a MenuStrip or Windows form.